hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/customize/ResourceServerConfigurationCustomizerDsl.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.hmpps.kotlin.customize
+
+@DslMarker
+annotation class ResourceServerConfigurationCustomizerDslMarker
+
+@ResourceServerConfigurationCustomizerDslMarker
+interface ResourceServerConfigurationCustomizerDsl {
+  @UnauthorizedRequestPathCustomizerDslMarker
+  fun unauthorizedRequestPaths(dsl: UnauthorizedRequestPathCustomizerDsl.() -> Unit): UnauthorizedRequestPathsCustomizer
+}
```

Comment:
New top level DSL to create a customizing bean for the library's resource server